generators/generator-base.js

```diff
@@ -41,7 +41,7 @@ util.inherits(Generator, yeoman.Base);
  */
 Generator.prototype.addElementToMenu = function (routerName, glyphiconName, enableTranslation) {
     try {
-        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.html';
+        var fullPath = CLIENT_MAIN_SRC_DIR + 'app/layouts/navbar/navbar.component.html';
```

Comment:
yes. we have the angularVersion variable on the generator context for that, so while generating an app or entity we will know which angular version is being used. its similar to how we do all other options today. So it will be a simple if else or even a fresh new method for ng2 if required. But this can be done later in any case